server/app.js

```diff
@@ -29,6 +34,7 @@ app.use(express.static(join(__dirname, "public")));
 
 app.use("/", indexRouter);
 app.use("/ping", pingRouter);
+app.use("/api/prospects", prospectRouter);
```

Comment:
As discussed in the stand-up, we should import our routers into `routes/index.js` and have that router `.use()` them